src/index.html

```diff
@@ -5,9 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
```

Comment:
HTML does not directly support SCSS files. You should compile your SCSS to CSS and include the CSS file in your HTML

Reply:
The task
❗️❗️❗️ In order to use SCSS delete the style.css file and change the <link> in the index.html to:
<link rel="stylesheet" href="./styles/index.scss">